azext_edge/edge/command_map.py

```diff
@@ -245,6 +246,159 @@ def load_iotops_commands(self, _):
         cmd_group.command("onvif", "create_onvif_asset_endpoint_profile", deprecate_info=cmd_group.deprecate(hide=True))
         cmd_group.command("opcua", "create_opcua_asset_endpoint_profile")
 
+    with self.command_group(
+        "iot ops ns",
+        command_type=namespace_resource_ops,
+        is_preview=True,
+    ) as cmd_group:
+        cmd_group.command("create", "create_namespace")
+        cmd_group.command("delete", "delete_namespace")
+        cmd_group.command("list", "list_namespaces")
+        cmd_group.show_command("show", "show_namespace")
+        cmd_group.command("update", "update_namespace")
+
+    with self.command_group(
+        "iot ops ns device",
+        command_type=namespace_resource_ops,
+    ) as cmd_group:
+        cmd_group.command("create", "create_namespace_device")
+        cmd_group.command("delete", "delete_namespace_device")
+        cmd_group.command("query", "query_namespace_devices")
+        cmd_group.show_command("show", "show_namespace_device")
+        cmd_group.command("update", "update_namespace_device")
+
+    with self.command_group(
+        "iot ops ns device endpoint",
+        command_type=namespace_resource_ops,
+    ) as cmd_group:
+        cmd_group.command("list", "list_namespace_device_endpoints")
+
+    with self.command_group(
+        "iot ops ns device endpoint inbound",
+        command_type=namespace_resource_ops,
+    ) as cmd_group:
+        cmd_group.command("list", "list_inbound_device_endpoints")
+        cmd_group.command("remove", "remove_inbound_device_endpoints")
+
+    with self.command_group(
+        "iot ops ns device endpoint inbound add",
+        command_type=namespace_resource_ops,
+    ) as cmd_group:
+        cmd_group.command("custom", "add_inbound_custom_device_endpoint")
+        cmd_group.command("media", "add_inbound_media_device_endpoint")
+        cmd_group.command("onvif", "add_inbound_onvif_device_endpoint")
+        cmd_group.command("opcua", "add_inbound_opcua_device_endpoint")
+
+    with self.command_group(
+        "iot ops ns asset",
+        command_type=namespace_resource_ops,
+    ) as cmd_group:
+        cmd_group.command("delete", "delete_namespace_asset")
+        cmd_group.command("query", "query_namespace_assets")
+        cmd_group.show_command("show", "show_namespace_asset")
+
+    with self.command_group(
+        "iot ops ns asset custom",
```

Comment:
I am doing some for looping in the streams + mgmt code base to minimize amount of duplicate code in command maps